src/ui/CategoryFacet/CategoryFacet.ts

```diff
@@ -152,42 +135,15 @@ export class CategoryFacet extends Component implements ICategoryFacet {
      */
     injectionDepth: ComponentOptions.buildNumberOption({ defaultValue: 1000, min: 0 }),
 
-    /**
-     * If the [`enableFacetSearch`]{@link CategoryFacet.options.enableFacetSearch} option is `true`, specifies the number of
-     * values to display in the facet search results popup.
-     *
-     * Default value is `15`. Minimum value is `1`.
-     */
-    numberOfResultsInFacetSearch: ComponentOptions.buildNumberOption({ defaultValue: 15, min: 1 }),
-
-    /**
-     * If the [`enableFacetSearch`]{@link CategoryFacet.options.enableFacetSearch} option is `true`, specifies the delay (in
-     * milliseconds) before sending a search request to the server when the user starts typing in the category facet search box.
-     *
-     * Specifying a smaller value makes results appear faster. However, chances of having to cancel many requests
-     * sent to the server increase as the user keeps on typing new characters.
-     *
-     * Default value is `100`. Minimum value is `0`.
-     */
-    facetSearchDelay: ComponentOptions.buildNumberOption({ defaultValue: 100, min: 0 }),
-
     /**
      * Whether to enable the **More** and **Less** buttons in the Facet.
      *
-     * See also the [`pageSize`]{@link CategoryFacet.options.pageSize} option.
+     * See also the [`numberOfValues`]{@link CategoryFacet.options.numberOfValues} option.
      *
      * Default value is `true`.
```

Comment:
Since you're modifying this comment, might as well remove this line and let TypeDoc generate it :)